src/Microsoft.AspNetCore.Mvc.Core/ProducesAttribute.cs

```diff
@@ -64,6 +64,14 @@ public ProducesAttribute(string contentType, params string[] additionalContentTy
 
         public MediaTypeCollection ContentTypes { get; set; }
 
+        public int StatusCode
+        {
+            get
+            {
+                return (int)HttpStatusCode.OK;
```

Comment:
This really seems like a great place to use `=>` and win valuable #PranavPoints /cc @pranavkm